torchci/rockset/commons/__sql/test_times_per_workflow_type.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
```

Comment:
Jut FYI, I randomly found and used this https://codebeautify.org/sqlformatter to format the initial version of my SQL query